src/sentry/static/sentry/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -174,7 +188,12 @@ export default class AbstractExternalIssueForm<
       : {};
 
   // Abstract methods.
-  getEndPointString = () => '';
+  handleReceiveIntegrationDetails = (_data: any) => {
```

Comment:
@mgaeta does this method need to be implemented by the child classes? If so, we should throw Error like in `getEndPointString`

Reply:
`handleReceiveIntegrationDetails` only used by the `TicketRuleModal` and should do nothing in the  `ExternalIssueForm`.

Reply:
@mgaeta Since we call this method in line95 in the `AbstractExternalIssueForm`, I'm leaning towards recommending that we make the child classes override this method with their preferred behaviour. That way, its explicit. Do you have an opinion on this?

Reply:
@NisanthanNanthakumar @mgaeta It's a fine line but I think this is OK to leave as is. Having the default behavior do nothing is how we handle a lot of functions on base classes that are implementing a side effect. It's different if it's core behavior you rely on (such as if you're expecting a return value). In those cases, it's better to throw an error in the abstract class and let the child classes override it.